src/SDKs/Blueprint/Blueprint.sln

```diff
@@ -0,0 +1,54 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
+VisualStudioVersion = 15.0.26430.4
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Management.Blueprint.Tests", "Blueprint.Tests\Management.Blueprint.Tests.csproj", "{E93AD1AE-BC72-45C9-90EB-878D45169599}"
+	ProjectSection(ProjectDependencies) = postProject
+		{6E64088C-3182-4E50-9450-2365508135F7} = {6E64088C-3182-4E50-9450-2365508135F7}
+	EndProjectSection
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.Management.Blueprint", "Management.Blueprint\Microsoft.Azure.Management.Blueprint.csproj", "{6E64088C-3182-4E50-9450-2365508135F7}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Net40-Debug|Any CPU = Net40-Debug|Any CPU
+		Net40-Release|Any CPU = Net40-Release|Any CPU
+		Net45-Debug|Any CPU = Net45-Debug|Any CPU
+		Net45-Release|Any CPU = Net45-Release|Any CPU
+		Portable-Debug|Any CPU = Portable-Debug|Any CPU
+		Portable-Release|Any CPU = Portable-Release|Any CPU
+	EndGlobalSection
+	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{E93AD1AE-BC72-45C9-90EB-878D45169599}.Net40-Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{E93AD1AE-BC72-45C9-90EB-878D45169599}.Net40-Debug|Any CPU.Build.0 = Debug|Any CPU
+		{E93AD1AE-BC72-45C9-90EB-878D45169599}.Net40-Release|Any CPU.ActiveCfg = Release|Any CPU
+		{E93AD1AE-BC72-45C9-90EB-878D45169599}.Net40-Release|Any CPU.Build.0 = Release|Any CPU
+		{E93AD1AE-BC72-45C9-90EB-878D45169599}.Net45-Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{E93AD1AE-BC72-45C9-90EB-878D45169599}.Net45-Debug|Any CPU.Build.0 = Debug|Any CPU
+		{E93AD1AE-BC72-45C9-90EB-878D45169599}.Net45-Release|Any CPU.ActiveCfg = Release|Any CPU
+		{E93AD1AE-BC72-45C9-90EB-878D45169599}.Net45-Release|Any CPU.Build.0 = Release|Any CPU
+		{E93AD1AE-BC72-45C9-90EB-878D45169599}.Portable-Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{E93AD1AE-BC72-45C9-90EB-878D45169599}.Portable-Debug|Any CPU.Build.0 = Debug|Any CPU
+		{E93AD1AE-BC72-45C9-90EB-878D45169599}.Portable-Release|Any CPU.ActiveCfg = Release|Any CPU
+		{E93AD1AE-BC72-45C9-90EB-878D45169599}.Portable-Release|Any CPU.Build.0 = Release|Any CPU
+		{6E64088C-3182-4E50-9450-2365508135F7}.Net40-Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
The only allowed configs are net452 and netstandard, please remove all other configs here